src/display/text_layer.js

```diff
@@ -132,9 +132,8 @@ var renderTextLayer = (function renderTextLayerClosure() {
     }
 
     var canvas = document.createElement('canvas');
-//#if MOZCENTRAL || FIREFOX || GENERIC
-    canvas.mozOpaque = true;
-//#endif
+    canvas.mozOpaque = typeof PDFJSDev === 'undefined' ||
+                       PDFJSDev.test('FIREFOX || MOZCENTRAL || GENERIC');
```

Comment:
Nit: this uses a slightly different form than the other `mozOpaque` occurrences, can we use the same one everywhere?